docs/security/configuration/security-admin.md

```diff
@@ -24,7 +24,7 @@ opendistro_security.authcz.admin_dn:
   - CN=kirk,OU=client,O=client,L=test,C=DE
 ```
 
-Do not use node certificates as admin certificates. The best practice is to keep these certificates separate. Also, do not use any whitespace between the parts of the DN.
```

Comment:
Hi @volkflo, is this true? My understanding is that you can technically do it, but that it's a really bad idea.

Reply:
Hi @aetter,
Thanks for the fast feedback. 
I can show you my thoughts about that:

`elasticsearch.yml`
```
...
  authcz:
    admin_dn:
     - "CN=docker-node,O=Scheer,L=Saarbruecken,ST=Saarland,C=DE"
  nodes_dn:
    - "CN=docker-node,O=Scheer,L=Saarbruecken,ST=Saarland,C=DE"
...
```
If I run the `securityadmin.sh` script it prints that output:

> [root@mynode elasticsearch]# chmod +x /usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh && /usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh -icl -nhnv -h my-host -cacert /usr/share/elasticsearch/config/root-ca.pem -cert /usr/share/elasticsearch/config/node.pem -key /usr/share/elasticsearch/config/node-key.pem -cd /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/
Open Distro Security Admin v7
Will connect to my-host:9300 ... done
Connected as CN=docker-node,O=Scheer,L=Saarbruecken,ST=Saarland,C=DE
ERR: Seems you use a node certificate which is also an admin certificate
     That may have worked with older Open Distro Security versions but it indicates
     a configuration error and is therefore forbidden now.

So, recording to that error message it is not possible to use the same certificates.